tags.json

```diff
@@ -2286,6 +2286,10 @@
     "weather",
     "light"
   ],
+  "sun-dim": [
+    "dim sun",
+    "dim"
```

Comment:
mind adding "brightness" to the tags here? maybe also add "brightness" to the regular "sun" icon as well